tools/automatic_query_fixer/src/test/java/com/google/cloud/bigquery/utils/queryfixer/QueryTokenProcessorTest.java

```diff
@@ -0,0 +1,30 @@
+package com.google.cloud.bigquery.utils.queryfixer;
+
+import com.google.cloud.bigquery.utils.queryfixer.entity.IToken;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class QueryTokenProcessorTest {
+
+  private QueryTokenProcessor createService() {
+    return new QueryTokenProcessor(new BigQueryParserFactory());
+  }
+
```

Comment:
This should be shared by all test cases in setup()?

Reply:
also createProcessor() :)

Reply:
Miss this comment. I will revise this in next commit

Reply:
I think you can make it a member in testSetup() so you don't have to call it for every test case